Hashes/tests/MD5.test.js

```diff
@@ -0,0 +1,37 @@
+import { MD5 } from '../MD5'
+
+describe('Testing MD5 function', () => {
+  it('should return the correct hash for "The quick brown fox jumps over the lazy dog"', () => {
+    const input = new TextEncoder().encode(
+      'The quick brown fox jumps over the lazy dog'
+    )
+
+    const hash = Array.from(MD5(input), (byte) =>
```

Comment:
Why not introduce a function `hexMD5` that does this? Duplicating this three times doesn't seem very viable.